### ISSUE TITLE
How to include multiple jobs (promotion) from the same company?

### ISSUE THREAD

**Issue by haozturk** [posquit0/Awesome-CV#297]
I could not figure out how to show multiple jobs (maybe promotion) from the same company? I think Linkedin has a neat structure for this purpose. Is there any opinion for this?

**Comment by Berkmann18** [comment]
The way I would do it by just having those several jobs even if the company/location is the same.

**Comment by haozturk** [comment]
Yes, it's a solution. But I'm trying not to exceed 1 page, therefore looking for a more compact form such as:

Company Name                                 Location
Position 1                                           Date for Position 1
Details for Position 1
Position 2                                           Date for Position 2
Details for Position 2

I am playing with the code for this purpose. If I succeed, I will share.

**Comment by Berkmann18** [comment]
Ah, I see! That would be nice indeed.

**Comment by ZarifS** [comment]
Hi I would also really appreciate this addition if anyone knows how to do it?

**Comment by haozturk** [comment]
I guess the following command works for this purpose:
```


\newcommand*{\cventrytwopositions}[8]{%
  \vspace{-2.0mm}
  \setlength\tabcolsep{0pt}
  \setlength{\extrarowheight}{0pt}
  \begin{tabular*}{\textwidth}{@{\extracolsep{\fill}} L{\textwidth - 4.5cm} R{4.5cm}}
      \entrytitlestyle{#2} & \entrylocationstyle{#3} \\
      \entrypositionstyle{#6} & \entrydatestyle{#7} \\
      \multicolumn{2}{L{\textwidth}}{\descriptionstyle{#8}} \\
      \entrypositionstyle{#1} & \entrydatestyle{#4} \\
      \multicolumn{2}{L{\textwidth}}{\descriptionstyle{#5}}
  \end{tabular*}%
}
```

After adding this new command definition to **awesome-cv.cls** file, you can create an instance of it, for example in the experience.tex file. Here is the order of the fields:

```
\cventrytwopositions
    {Position1}
    {Company}
    {Location}
    {Date for position 1}
    {
      Details for position 1
    }
    {Position2}
    {Date for position 2}
    {
      Details for position 2
    }
```

Of course, order of the arguments could be altered. Let me know if this works or not, so that we can improve.

**Comment by mircoparello** [comment]
Not the best solution, but I fixed simply by adding two cventries without writing the organization and location of the oldest one.
Something like

`  \cventry
    {Job title after promotion} 
    {Corporation} 
    {Location} 
    {August 2019 - now}
    {
      \begin{cvitems} 
        \item {Responsibilities}
        \end{cvitems}
 	}


  \cventry
    {Job title before your promotion} % Job title
    {} % Organization EMPTY
    {} % Location EMPTY
    {somewhere in the past - a while ago} % Date(s)
    {
      \begin{cvitems} % Description(s) of tasks/responsibilities
        \item {Old responsibilities}
        \end{cvitems}
 	}`

**Comment by zzamboni** [comment]
I do the same as @mircoparello, but I defined a new command `cvsubentry` for this purpose:
```latex
% Sub-job entry - assumes the organization and place are specified in a preceding cventry
% Usage: \cvsubentry{<position>}{<date>}{<description>}
\newcommand*{\cvsubentry}[3]{\cventry{}{}{#1}{#2}{#3}}
```

**Comment by Berkmann18** [comment]
I tried using @posquit0's `cvsubentry` thing as he used in https://github.com/posquit0/Awesome-CV/blob/master/examples/cv/experience.tex but XeLaTeX doesn't like that ("! Extra alignment tab has been changed to \cr. <recently read> \endtemplate").

So for now, I'm doing the same as @mircoparello which looks good enough.

**Comment by adnan-azmat** [comment]
This works if you are promoted in the same location, what if I was promoted in the same company but to a different work location?

**Comment by OJFord** [comment]
@adnan-azmat This is how I solve it in my own CV:
```tex
% as \cventry, but squashed up to meet more recent role above; no company/location.
\newcommand{\cventryprevrole}[3]{
  \vspace{-3.0mm}
  \setlength\tabcolsep{0pt}
  \setlength{\extrarowheight}{0pt}
  \begin{tabular*}{\textwidth}{@{\extracolsep{\fill}} L{4.5cm} L{\textwidth - 4.5cm}}
    \entrydatestyle{#2} & \entrypositionstyle{#1} \\
    & \descriptionstyle{#3}
  \end{tabular*}
}
```
https://github.com/OJFord/curriculum-vitae/blob/0989831fb4810d1520bee67cbd93553458efafd5/cv.cls#L82-L91

This could easily be adapted to include (I deliberately removed it for my use) location. For comparison, here's `\cventry`:
https://github.com/posquit0/Awesome-CV/blob/db809982227ba6f24f2d0ec1c4f3893e99db2f41/awesome-cv.cls#L657-L670

I've been meaning to read through others' suggestions & implementations here and elsewhere before coming up with something to include in Awesome-CV itself to address this.
